ros_ws/src/common/config.h

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file config.h
+ * @struct ConfigValues
+ * @brief Contains the struct ConfigValues.
+ *
+ * To use ConfigValues to access a config value:
+ * add "ConfigValues config;" to the class constructor and
+ * add "config = initConfigs()" where ever you need to use it
+ * @see initConfigs()
+ */
+
+#ifndef CONFIG_H_
+#define CONFIG_H_
+
+
+#include <ros/ros.h>
+#include "constants.h"
+
+
+struct ConfigValues
```

Comment:
This struct may end up getting really big when everything is said and done.  A hash table or map might be a better solution.  @islamelnabarawy What do you think?